src/crt.h

```diff
@@ -11,6 +11,9 @@
 #include "api.h"
 /* clang-format on */
 
+enum aws_napi_log_subject { AWS_LS_CRT = 0x900 };
```

Comment:
what's this, esp. the value? (AWS_LOG_SUBJECT_BEGIN_RANGE, AWS_LOG_SUBJECT_END_RANGE)

Reply:
I copied this from node to get moving, as you can tell by its shitty name, forgot to go back and fix it.